src/lib/Preview.js

```diff
@@ -4,7 +4,7 @@ import EventEmitter from 'events';
 import cloneDeep from 'lodash/cloneDeep';
 import throttle from 'lodash/throttle';
 /* eslint-enable import/first */
-import api from './api';
+import Api from './api';
```

Comment:
I think the preference is that acronyms be uppercased

Reply:
What if we are trying to denote that it is a constructor?